flow/otel_metrics/attributes.go

```diff
@@ -1,8 +1,20 @@
 package otel_metrics
 
 const (
-	PeerNameKey      string = "peerName"
-	SlotNameKey      string = "slotName"
-	FlowNameKey      string = "flowName"
-	DeploymentUidKey string = "deploymentUID"
+	PeerNameKey         string = "peerName"
+	SlotNameKey         string = "slotName"
+	FlowNameKey         string = "flowName"
+	DeploymentUidKey    string = "deploymentUID"
+	ErrorClassKey       string = "errorClass"
+	InstanceStatusKey   string = "instanceStatus"
+	WorkflowTypeKey     string = "workflowType"
+	BatchIdKey          string = "batchId"
+	SourcePeerType      string = "sourcePeerType"
+	DestinationPeerType string = "destinationPeerType"
```

Comment:
```suggestion
	PeerNameKey         = "peerName"
	SlotNameKey         = "slotName"
	FlowNameKey         = "flowName"
	DeploymentUidKey    = "deploymentUID"
	ErrorClassKey       = "errorClass"
	InstanceStatusKey   = "instanceStatus"
	WorkflowTypeKey     = "workflowType"
	BatchIdKey          = "batchId"
	SourcePeerType      = "sourcePeerType"
	DestinationPeerType = "destinationPeerType"
```
No need to say `string`. & if you do prefer the explicit typing that need only be specified on first declaration. But that's more for if you decided to go with `type AttributeKey string`

Reply:
ik, just my autocomplete kicked in will refactor